docs/book/src/developer/tilt.md

```diff
@@ -325,7 +325,7 @@ Custom values for variable substitutions can be set using `kustomize_substitutio
 ```yaml
 kustomize_substitutions:
```

Comment:
@killianmuldoon Can we please also:
* Update `v1.4-to-v1.5.md`
* Update the min kind version in tilt.md
* Update the min kind version in quick-start.md

Regarding the docs, I don't want to keep the old kind version there as we're not testing it

Reply:
https://github.com/kubernetes-sigs/cluster-api/pull/8699